app/views/layouts/header.html.erb

```diff
@@ -0,0 +1,7 @@
+<nav class="navbar navbar-light bg-light">
+  <!--- giving profiles needs to be two lines - do we need a logo or should we do this in text? -->
```

Comment:
Good Question! Maybe @uuunk can answer that.

Reply:
Also we shouldn't commit a comment about questions we are having in the code

Reply:
I think it's okay to leave such a comment, but it should perhaps be listed in the PR, or created as a story on Trello.

Reply:
I'm not a huge stickler on a comments as this, but I think I should elaborate just so that @emilyspieler is aware of why one might think to not include comments like this.

Generally, the industry standard for commenting is when you are:

**Documenting** - You have a library or an API that needs to specify the ins and outs of your public methods
**Clarifications** -  You are implementing logic that is required for the feature, but hard for other developers to understand due to unknown side-effects, etc.

Outside of these situations, it's more often discouraged to add comments as can potentially lead to lies if the logic you begin to iterate upon strays away from the comment you have made to that logic on its initial implementation. Also, it can be seen as unprofessional depending on how "strict" or "opinionated" people can get.

Reply:
Yeah i'm not too much of a stickler either, but I agree that the vast majority of comments should be one of the 3 @sanghun89 mentioned. Asking for feedback is best left to a pair session/slack convo/etc. If it's something that you know needs to be fixed or improved but going with as is, then a TODO is best.

And to answer the question in the comment @emilyspieler , we should be able to do that with text for now, maybe can use a logo in the future. Think you can use a <br> to put it on two lines.